### PR TITLE
[chore] migrate custom tax

### PR DESCRIPTION
**Pull request by garguelles**
# Changes

- Grocery tax plugin

### CODE REVIEW

**Review by ajmiclat — APPROVED** [review]

File: saleor/plugins/uttax/plugin.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            price_with_sale=flat_tax(
+                previous_value.price_with_sale, Decimal(0.03)
+            ).quantize(),
+        )
+        return line_total
+
+
+def _validate_checkout(checkout: "Checkout", lines: Iterable["CheckoutLine"]) -> bool:
+    """Validate the checkout object if it is ready to generate a request to avatax."""
+    if not lines:
```
Comment (jsnbuchanan) [inline saleor/plugins/uttax/plugin.py:60]: @garguelles I think the doc is wrong. This does not leverage "Avatax," does it?

File: saleor/plugins/uttax/plugin.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    is_shipping_required = checkout.is_shipping_required()
+    address = shipping_address or checkout.billing_address
+    return _validate_address_details(
+        shipping_address, is_shipping_required, address, checkout.shipping_method
+    )
+
+
+def _validate_order(order: "Order") -> bool:
+    """Validate the order object if it is ready to generate a request to avatax."""
+    if not order.lines.exists():
```
Comment (jsnbuchanan) [inline saleor/plugins/uttax/plugin.py:73]: This doc too.

**Review by jsnbuchanan — COMMENTED** [review]
@garguelles small nit on a couple of the doc strings.

File: saleor/plugins/uttax/plugin.py (hunk 1 above)
Comment (garguelles) [reply saleor/plugins/uttax/plugin.py:60]: correct. will remove the avatax word

**Pull request merged**